config/settings.yml

```
@@ -20,5 +20,6 @@ answer:
     max_1000: 1000
 paging:
   subjects_per_page_15: 15
+  questions_per_page_10: 10
```

```suggestion
  per_page_10: 10
```
Ri là đủ để còn tái sử dụng